pkg/kotscli/kotscli.go

```diff
@@ -134,7 +137,8 @@ type VeleroConfigureOtherS3Options struct {
 }
 
 func VeleroConfigureOtherS3(opts VeleroConfigureOtherS3Options) error {
-	kotsBinPath, err := goods.MaterializeInternalBinary("kubectl-kots")
+	materializer := goods.NewMaterializer(defaults.EmbeddedClusterHomeDirectory())
```

Comment:
shouldn't this use the runtimeconfig?

Reply:
this pr only attempts to consolidate the directories. not allow them to be overwritten. i will do that in a follow up